### ISSUE TITLE
Print(N) crash after N.clear()

### ISSUE THREAD

**Issue by pcchen** [kaihsin/Cytnx#166]
If I try to print a network N after N.clear(), it crashes the kernel....

````
N = cytnx.Network()
N.FromString(["A: 0,1,2",\
              "B: 0,3,4",\
              "C: 5,1,6",\
              "D: 5,7,8",\
              "TOUT: 2,3,4;6,7,8",\
              "ORDER: (A,B),(C,D)"\
             ])
 
print(N)
N.clear()
print(N)
````

**Comment by j9263178** [comment]
Fixed in [27b0175](https://github.com/kaihsin/Cytnx/commit/27b0175c2dab413966f80a4533e78ff789c08dc9).
Now it should output like:
```
==== Network ====
      Empty 
```

**Comment by pcchen** [comment]
This is to confirm that [27b0175] fixes this issue.
